src/components/Editor.js

```diff
@@ -34,6 +34,7 @@ class Editor extends Component {
         highlightActiveLine={false}
         className="config-editor"
         fontSize={14}
+        scrollMargin={[15, 15, 15, 15]}
```

Comment:
this is to simulate padding in the `ace-editor` module. It is activated by an `onScroll` call